ketcher-autotests/tests/Indigo-Tools/Layout/layout.spec.ts

```diff
@@ -44,4 +45,13 @@ test.describe('Indigo Tools - Layout', () => {
     await selectTopPanelButton(TopPanelButton.Layout, page);
     await takeEditorScreenshot(page);
   });
+
+  test('After applying Layout, the structure does not disappear and can be interacted with', async ({
+    page,
+  }) => {
+    // Related Github issue: https://github.com/epam/ketcher/issues/3208
+    await openFileAndAddToCanvas('Molfiles-V2000/sbs_48.mol', page);
+    await selectTopPanelButton(TopPanelButton.Layout, page);
+    await takeEditorScreenshot(page);
```

Comment:
Hmm, might need to wait for Indigo here?

Reply:
Is it necessary to wait for some time after pressing the layout button to check if the structure has disappeared, if the button immediately arranges structures in place?